tests/std/tests/P0896R4_common_iterator/test.cpp

```diff
@@ -194,6 +194,37 @@ bool test_operator_arrow() {
     return true;
 }
 
+// After P2259, common_iterator supports "copyable but not equality_comparable" iterators, which combination
+// test::iterator does not provide (I don't think this is a combination of properties that any real iterator will ever
+// exhibit). Whip up something so we can test the iterator_category metaprogramming.

```

Comment:
It has always supported that? P2259 simply makes it support such types even if postfix++ returns `void` (which makes it fail to satisfy _`cpp17-iterator`_.